### ISSUE TITLE
ADD sysb.AI ssh Tunnel

### ISSUE THREAD

**Issue by An0nUD4Y** [DarkSecDevelopers/HiddenEye#520]
Just came to know about sysb.AI 
So to decrease the chances of tunnel failure , should i add this tunnel as an extra option.

Please leave your suggestions...

**Comment by An0nUD4Y** [comment]
I will also add https://tunnel.staqlab.com/ as an option.

**Comment by sTiKyt** [comment]
Don't add suspicious one's for now
We'll have to be sure about them before we add

**Comment by sTiKyt** [comment]
More doesn't mean better

**Comment by usama7628674** [comment]
@sTiKyt Good news,Serveo is back online.

**Comment by bluewarrior2457** [comment]
Yeah usama it's great. Before that I was using burrow.io... It's cool to..... But somehow the page was hacked and it was redirecting me to some shitty song😂

**Comment by darkmidus** [comment]
I will add some tunnels when I get some spare time.

**Comment by sTiKyt** [comment]
@bluewarrior2457 yeah, that's why i want us to get slower and don't add those so fast..
